Capfile

```diff
@@ -16,21 +16,13 @@ install_plugin Capistrano::SCM::Git
 #   https://github.com/capistrano/rbenv
 #   https://github.com/capistrano/chruby
 #   https://github.com/capistrano/bundler
-#   https://github.com/capistrano/rails
```

Comment:
This app does not use capistrano-rails so I'm not convinced we need all this cruft hanging around. If we ever add DB migrations or assets to DIA, we can always add this back.